src/containers/Footer/index.jsx

```diff
@@ -1,130 +1,16 @@
-import React, { useState } from 'react'
+import React from 'react'
 import PropTypes from 'prop-types'
 import { useTranslation } from 'react-i18next'
 import { connect } from 'react-redux'
 import { bindActionCreators } from 'redux'
-import classnames from 'classnames'
-
-import { analytics, ANALYTIC_TYPES } from '../shared/utils'
 import { updateLanguage } from '../App/actions'
 
 import packageConfig from '../../../package.json'
 import { ReactComponent as Logo } from '../shared/images/XRPLedger.svg'
-import iconDownArrow from '../shared/images/down_arrow.svg'
-import iconCheck from '../shared/images/checkmark_small_green.png'
-import flagUSA from '../shared/images/flag_usa.png'
```

Comment:
Should we just delete all these images too and maybe translation files that are never updated?

Reply:
I'm down to remove the images, but I think the translation files should stay - if/when we do decide to support multiple languages, it'll be a useful starting point.